docs/python/configuration.md

```diff
@@ -0,0 +1,240 @@
+advantage of your specific environment.
+
+Have configuration needs not addressed here? Give us a holler at
+[hi@yourbase.io][hi].
+
+- Common Settings
+  - [`YOURBASE_ACCEPT_TOS`](#YOURBASE_ACCEPT_TOS)
+  - [`YOURBASE_ACTIVE_COHORT`](#YOURBASE_ACTIVE_COHORT)
+  - [`YOURBASE_COHORT_COUNT`](#YOURBASE_COHORT_COUNT)
+  - [`YOURBASE_LICENSE_KEY`](#YOURBASE_LICENSE_KEY)
+  - [`YOURBASE_OBSERVATION_MODE`](#YOURBASE_OBSERVATION_MODE)
+  - [`YOURBASE_REMOTE_CACHE`](#YOURBASE_REMOTE_CACHE)
+- Uncommon settings
+  - [`YOURBASE_AWS_ACCESS_KEY_ID`](#YOURBASE_AWS_ACCESS_KEY_ID)
+  - [`YOURBASE_AWS_SECRET_ACCESS_KEY`](#YOURBASE_AWS_SECRET_ACCESS_KEY)
+  - [`YOURBASE_DEBUG`](#YOURBASE_DEBUG)
+  - [`YOURBASE_DISABLE`](#YOURBASE_DISABLE)
+  - [`YOURBASE_IGNORE_LOCAL_CACHE`](#YOURBASE_IGNORE_LOCAL_CACHE)
+  - [`YOURBASE_SYNC_DIRTY`](#YOURBASE_SYNC_DIRTY)
+  - [`YOURBASE_TELEMETRY`](#YOURBASE_TELEMETRY)
+  - [`YOURBASE_TIMID`](#YOURBASE_TIMID)
+  - [`YOURBASE_WORKDIR`](#YOURBASE_WORKDIR)
+
+## Common Settings
+
+### `YOURBASE_ACCEPT_TOS`
+
+- **Type:** bool-ish (`0`, `false`, `off`, `1`, `true`, `on`)
+- **Default:** off
+
+When set, YourBase will consider the terms of service permanently accepted for
+your organization, and will not output terms of service agreement prompts or
+info messages.
+
+### `YOURBASE_ACTIVE_COHORT`
+
+- **Type:** integer in the range `[1, $YOURBASE_COHORT_COUNT]`
+- **Default:** 1
+
+When set alongside [`YOURBASE_COHORT_COUNT`](#YOURBASE_COHORT_COUNT), tells
+YourBase the cohort ID to run. Used for sharded or otherwise parallelized test
+suites.
+
+See [`YOURBASE_COHORT_COUNT`](#YOURBASE_COHORT_COUNT) for more information.
+
+### `YOURBASE_COHORT_COUNT`
+
+- **Type:** integer
+- **Default:** 1
+
+When set alongside [`YOURBASE_ACTIVE_COHORT`](#YOURBASE_ACTIVE_COHORT), tells
+YourBase how many cohorts tests should be split into. Used for sharded or
+otherwise parallelized test suites.
+
+This pair of settings lets YourBase work with your existing sharding or
+parallelization setup. You likely already have a test splitting solution in
+place whose job is to slice up your pool of tests so that each shard or process
+runs their fair share of tests.
+
+However, most test splitting solutions are not sticky; tests "jump" shards as
+the pool changes. The effectiveness of YourBase acceleration scales with test
+stickiness, so these solutions are not recommended for use with YourBase.
+
+As a convenience, YourBase provides a built-in sticky test splitting
+implementation. To use it, first remove your existing test splitting tools. It
+should appear as if each shard will run the entire test suite.
+
+Then, set `YOURBASE_ACTIVE_COHORT` to the ID of the current shard or process
+(starting from 1), and set `YOURBASE_COHORT_COUNT` to the total number of shards
+or processes. Under the hood, YourBase will just-in-time deselect tests that
+aren't in the current cohort. Each test will stick to its assigned shard for
+life.
```

Comment:
```suggestion
life or until the value of YOURBASE_ACTIVE_COHORT is changed.
```

Reply:
There is a subtlety with the consistency of the cohorts that assignment depends on both the active cohort and the total cohorts.  So it's until/unless `YOURBASE_ACTIVE_COHORT` **or**  `YOURBASE_COHORT_COUNT` are changed.